src/components/parse-html.js

```diff
@@ -27,7 +27,7 @@ export const getStyleObjectFromString = (styles) => {
     if (!property) return
 
     const formattedProperty = formatStringToCamelCase(property.trim())
-    style[formattedProperty] = value.trim()
+    style[formattedProperty] = value ? value.trim() : null
```

Comment:
Was getting a TypeError when value was null when I added the markdown for an ![image](image.png). This got me passed that error.

![Screen Shot 2021-04-23 at 9 32 26 AM](https://user-images.githubusercontent.com/749603/115886828-ef8a8300-a416-11eb-8c87-e561e5d4a7ce.png)